### PR TITLE
add namespace

### PR DESCRIPTION
**Pull request by wyyxd2017**
[add namespcae](https://github.com/wyyxd2017/docs/commit/852ba29592814de4ecdf436b0d0b7a69270877ec)

### CODE REVIEW

File: docs/eventing/getting-started.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ demonstrate how you can configure your event producers to target a specific cons
      kind: Deployment
      metadata:
        name: hello-display
+       namespace: event-example
```
Comment (csantanapr) [inline docs/eventing/getting-started.md:71]: The namespace is being handled in the command line `kubectl -n event-example` do not add in the yaml. 
Doing in the command line is more flexible as allows the resource to be deployed to another namespace.
/hold

**Review by csantanapr — CHANGES_REQUESTED** [review]
/hold

File: docs/eventing/getting-started.md (hunk 1 above)
Comment (hatowang) [reply docs/eventing/getting-started.md:71]: ok，thanks

**Pull request closed without merging**